components/checkout/checkout-cart.tsx

```diff
@@ -1,24 +1,28 @@
 import { getCart } from "@/lib/sfcc";
```

Comment:
Im not totally satisfied with this approach - but needed to reuse the summary portion of this for order confirmation. Maybe make it a single component again, but do `getOrder` and `getCart` - will adjust.